src/GetPlasterTemplate.ps1

```diff
@@ -0,0 +1,86 @@
+. $PSScriptRoot\GetModuleExtension.ps1
+
+function Get-PlasterTemplate {
+    [CmdletBinding()]
+    param(
+        [Parameter(Position=0,
+                   ParameterSetName="Path",
+                   ValueFromPipeline=$true,
+                   ValueFromPipelineByPropertyName=$true,
+                   HelpMessage="Specifies a path to a folder containing a Plaster template or multiple template folders.  Can also be a path to plasterManifest.xml.")]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Path,
+
+        [Parameter(ParameterSetName="Path",
+                   HelpMessage="Indicates that this cmdlet gets the items in the specified locations and in all child items of the locations.")]
+        [switch]
+        $Recurse,
+
+        [Parameter(Position=0,
+                   Mandatory=$true,
+                   ParameterSetName="InstalledModules",
+                   HelpMessage="Initiates a search for Plaster templates inside of installed modules.")]
+        [switch]
+        $IncludeModules
```

Comment:
What do you think about `IncludeInstalledModules` instead of `IncludeModules`?  Seems clearer even if more verbose.